.github/workflows/publishing.yml

```diff
@@ -219,8 +219,7 @@ jobs:
             if [[ "$staging_branch" =~ ^bot\/.*$ ]]; then
               git push origin --delete $staging_branch
             else
-              echo "Unexpected staging branch."
-              exit 1
+              echo "::error::Unexpected staging branch." && exit 1
```

Comment:
Should this echo to stderr?

Reply:
This syntax is a GitHub specific format that will make this show as an error message in the pipeline. I would need to try out if this still works when piped to the error stream. That said, if we update this, we should probably update it in all workflows (suggest to defer that for a separte PR).

Reply:
I see it is a .yml file, so sounds good.